packages/sanity/src/core/studio/components/navbar/search/components/searchResults/SearchResults.tsx

```diff
@@ -101,6 +101,7 @@ export function SearchResults({inputElement}: SearchResultsProps) {
                     paddingY={1}
                     ref={setSearchCommandList}
                     renderItem={renderItem}
+                    data-testid="search-results"
```

Comment:
Nitpick: The props on this `CommandList` are alphabetized – so it would be nice to follow that pattern with the added test id attribute!

Reply:
Thanks for the feedback, @hermanwikner ! Unfortunately, `data-testid` is a [Playwright convention](https://playwright.dev/docs/locators#locate-by-test-id), and I'm not sure if there's an easy way around it. If there's somewhere else in the DOM tree you'd like me to try that would be less intrusive, please let me know.

Reply:
Thanks for the feedback Herman! (apologies for my misread earlier)